### DISCUSSION TITLE
zh/guide/drivers/chaoxing

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#309] [Announcements]
# zh/guide/drivers/chaoxing

超星星小组盘也是隶属于超星星，登录超星星后在左侧侧边栏下方能看到一个小组 官网链接：https://i.chaoxing.com 用户名 密码 超星星的帐号密码 Bbsid 按照下面示例流程获取 根文件夹ID 默认为-1，其它子文件夹ID获取方式看下图示例 在请求中搜索关键词getResourceList，为了方便我们可以直接搜索 list也可以看到，...

https://alist.nn.ci/zh/guide/drivers/chaoxing.html

**Comment by liaolia333** [comment]
大佬啊，这个没找到添加的入口，是webdav吗？好像不对啊

**Reply by liaolia333** [reply]
无法使用

**Comment by renjiu13** [comment]
请问显示：failed get objs: failed to list objs: error code is:0，是什么意思

**Comment by foxdodo** [comment]
试了一下，竟然会ban号🤣

**Comment by shaonianshuaida** [comment]
求大佬更新，无法上传了

**Comment by ipme** [comment]
### 提示错误
``` html
failed get objs: failed to list objs: Get "https://passport2-api.chaoxing.com/views/error/passport403.html": stopped after 10 redirects
```

### 不知道是什么情况了

**Comment by pipinian** [comment]
failed get objs: failed to list objs: json: cannot unmarshal string into Go struct field .list.content.uploadDate of type int64
打不开了

**Comment by izhuolin** [comment]
添加小组盘设置成功 进入目录 提示
failed get objs: failed to list objs: json: cannot unmarshal string into Go struct field .list.content.uploadDate of type int64

**Reply by gsessylu** [reply]
上传文件返回405错误

**Comment by gsessylu** [comment]
GIN] 2024/04/27 - 00:22:47 | 405 |    581.3706ms |       127.0.0.1 | PUT      "/dav/超星网盘/output.csv"
[31mERRO[0m[2024-04-27 00:22:48] failed put /超星网盘: Time.UnmarshalJSON: input is not a JSON string

**Reply by mxl101** [reply]
failed get objs: failed to list objs: json: cannot unmarshal string into Go struct field .list.content.uploadDate of type int64

**Comment by liaolia333** [comment]
2024年8月2日实测无法使用

**Comment by MJX11121** [comment]
这个、没有302几乎就没有用
